x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/withContextComponents/index_mapping_with_context.tsx

```diff
@@ -6,14 +6,14 @@
  */
```

Comment:
Moved files that uses `withContext` & embeddables to new folder [withContextComponents](https://github.com/elastic/kibana/tree/93b522266cc2f35f5f09e505d500df0fc65807a3/x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/withContextComponents)